app/modules/asset_groups/schemas.py

```diff
@@ -27,11 +27,15 @@ class DetailedAssetGroupSightingSchema(BaseAssetGroupSightingSchema):
     Detailed Asset_group_sighting schema exposes all useful fields.
     """
 
+    completion = base_fields.Function(AssetGroupSighting.get_completion)
```

Comment:
> the magic of how this lambda allows you to get at the object to call the method is a bit of a mystery
> I have no idea if this is a really clever idea or a bad hack :)
> Any comments?

@hwindsor The difference between `AssetGroupSighting.get_compeltion` and `lambda obj: obj.get_completion()` is:

```
AssetGroupSighting.get_completion(asset_group_sighting)
```

and

```
asset_group_sighting.get_completion()
```

I think `lambda obj: obj.get_completion()` is probably better...?

`lambda obj: obj.get_completion()` is basically:

```
def f(obj):
    return obj.get_completion()
```

It's not a clever idea or bad hack, it's just a normal way of creating a callable